config/csi/kustomization.yaml

```diff
@@ -2,44 +2,5 @@
 namespace: ibm-spectrum-scale-csi
 
 resources:
-- github.com/IBM/ibm-spectrum-scale-csi.git?ref=v2.5.1
+- github.com/IBM/ibm-spectrum-scale-csi.git//operator/config/overlays/cnsa?ref=dev
```

Comment:
This needs to point away from `dev` reference

Reply:
Yeah...but I'm not sure if CSI has cut their tag yet.

Reply:
They have! Will update now. Thank you!